### DISCUSSION TITLE
How to use this provider?

### DISCUSSION THREAD

**Discussion by ublanke** [fleaflet/flutter_map_cancellable_tile_provider#5] [Q&A]
Hi all,

I am a little confused how to use the provider. Intuitively, I thought this should be enough:

 return TileLayer(
      tileProvider: CancellableNetworkTileProvider(),
      urlTemplate: <some url>,
    );
  }

But I still get the message:

│ 💡 flutter_map
│ 💡 Consider installing the official 'flutter_map_cancellable_tile_provider' plugin for improved
│ 💡 performance on the web.
│ 💡 See https://pub.dev/packages/flutter_map_cancellable_tile_provider for more info.

What am I missing?

Thanks,
Ulf

**Comment by JaffaKetchup** [comment]
Hi @ublanke,
This is logged when a `TileLayer` is constructed with a `NetworkTileProvider` on web. Are you constructing a `TileLayer` anywhere else?
